{"data":null,"cause":{},"code":422,"name":"AssertionFailureError","status":42206,"message":"Couldn't resolve host name","readableMessage":"AssertionFailureError: Couldn't resolve host name"}